server/data/userClient.js

```diff
@@ -16,35 +15,15 @@ module.exports = {
     return []
   },
 
-  async getIncompleteRoUsers() {
+  async getCasesRequiringRo() {
     const query = {
-      text: `select
-               timestamp as sent_timestamp,
-               details -> 'bookingId' as booking_id,
-               details -> 'submissionTarget' -> 'com' -> 'deliusId' as sent_staffcode,
-               details -> 'submissionTarget' -> 'com' -> 'name' as sent_name,
-               (s.staff_id is not null) as mapped,
-               s.auth_onboarded,
-               s.first_name, s.last_name, s.nomis_id
-             from (
-                    select *,
-                           row_number() over (
-                             partition by (a.details -> 'submissionTarget' -> 'com' -> 'deliusId')
-                             order by a.timestamp ASC
-                             ) as rownbr
-                    from audit a
-                           left join staff_ids s
-                                     on s.staff_id = (a.details-> 'submissionTarget' -> 'com' ->> 'deliusId')
-                    where a.action = 'SEND'
-                      and a.details @> '{ "transitionType": "caToRo" }'
-                  ) s
-             where rownbr = 1`,
+      text: `select booking_id from licences where stage in ('ELIGIBILITY', 'PROCESSING_RO')`,
     }
 
     const { rows } = await db.query(query)
 
     if (rows) {
-      return rows.map(convertIncompleteUserPropertyNames)
+      return rows.map(r => r.booking_id).filter(r => r)
```

Comment:
I don't really understand the intention for filter here.. Can you explain?

Reply:
Just to remove empty rows but now I realise that shouldn't happen from this query anyway. Will remove

Reply:
I see perhaps you can do it this way to show your intention by writing it as follows:
```
(r => r.booking_id).filter(Boolean)
```

This way all falsy values are cast into booleans?